services/dashboard/styles/bootstrap-override.css

```diff
@@ -155,7 +155,7 @@ html, body {
 
 /* Specify the size of modal dialog */
 .modal-md {
-  width: 520px;
+  margin-top: 30px;
```

Comment:
The margin is consistent with bootstrap modal dialog style.
Check:
![margin](https://cloud.githubusercontent.com/assets/2595532/12382014/fdecb2f2-bdcf-11e5-8378-e2489b6447d1.png)